src/core_modules/capture-core/components/Pages/StageEvent/StageEventListPage.component.js

```diff
@@ -0,0 +1,27 @@
+// @flow
+import React from 'react';
+import type { ComponentType } from 'react';
+import withStyles from '@material-ui/core/styles/withStyles';
+import { spacersNum } from '@dhis2/ui';
+import { compose } from 'redux';
+import type { Props } from './StageEventListPage.types';
+import { StageEventListComponent } from './StageEventList/StageEventList.component';
+import { withErrorMessageHandler } from '../../../HOC';
+
+const getStyles = ({ }) => ({
```

Comment:
You didn't deconstruct any variable here, are the parenthesis `{ } `needed? Thanks!